lms/templates/verify_student/make_payment_step_ab_testing.underscore

```diff
@@ -0,0 +1,108 @@
+<div id="wrapper-review" tab-index="0" class="wrapper-view make-payment-step">
+   <div class="review view">
+    <% if (!isActive ) { %>
+        <h2 class="page-title">
+            <%= _.sprintf(
+                gettext("Activate your account before upgrade to a  %(courseModeSlug)s certificate"),
+                { courseModeSlug:courseModeSlug }
+            ) %>
+        </h2>
+    <% } else if ( !upgrade ) { %>
+      <h2 class="page-title">
+        <%= _.sprintf(
+            gettext( "You are enrolling in: %(courseName)s"),
+            { courseName: '<span class="course-title">' + courseName + '</span>' }
+          ) %>
+      </h2>
+    <% } else { %>
+      <h2 class="page-title">
+        <%= _.sprintf(
+            gettext( "Upgrade to Verified Certificate: %(courseName)s"),
+            { courseName: '<span class="course-title">' + courseName + '</span>' }
+          ) %>
+      </h2>
+    <% } %>
+
+    <% if ( !isActive ) { %>
+        <p>
+            <%- gettext("Prior to completing this course purchase, you must activate your account.") %>
+            <%- gettext("Check your email for the activation link.") %>
```

Comment:
Before you upgrade to a certificate track, you must activate your account. 
Check your email for an activation message.